x-pack/plugins/entity_manager/server/lib/auth/privileges.ts

```diff
@@ -54,7 +58,7 @@ export const canDisableEntityDiscovery = async (client: ElasticsearchClient) =>
   );
 };
 
-export const entityDefinitionRuntimePrivileges = {
+export const entityDefinitionRuntimePrivileges = (sourceIndices: string[]) => ({
```

Comment:
It looks like we're not using the passed in argument anywhere here?

Reply:
good catch. fixed in https://github.com/elastic/kibana/pull/198695/commits/0abac645c7e74c4a9e030f2915d7e1d41eb85ba1